Admin/AdminHelper.php

```diff
@@ -284,13 +285,11 @@ public function getElementAccessPath($elementId, $entity)
                     $partReturnValue .= $dot.$id;
                     $dot = '.';
                     $id = '';
-                } else {
-                    $dot = '';
                 }
             }
 
-            if ($dot !== '.') {
-                throw new \Exception(sprintf('Could not get element id from %s Failing part: %s', $elementId, $subValue));
+            if (!empty($id)) {
```

Comment:
No problem, I will try to explain.

The else was just for throwing the exception and it's the part of the code that makes the issue appear.

And I changed the condition to be consistent with the exception. When the id is not empty is because it failed at finding the right path, and the part failing is exactly id, not subValue like before.